config/root/clusterworkspacetype-team.yaml

```diff
@@ -7,7 +7,9 @@ spec:
   defaultChildWorkspaceType:
     name: Universal
     path: root
-  allowAnyChildWorkspaceTypes: true
-  allowedParentWorkspaceTypes:
+  allowedChildren:
+    any: true
```

Comment:
isn't (can't) this be defaulted?

Reply:
Isn't it? Defaults to false, keeps the behavior we had in the past.